ruma-federation-api/src/event/get_missing_events/v1.rs

```diff
@@ -38,7 +39,7 @@ ruma_api! {
     #[derive(Default)]
     response: {
         /// The missing PDUs.
-        pub events: Vec<Pdu>
+        pub events: Vec<Raw<PduStub>>
```

Comment:
I'm not sure replacing `Pdu` with `PduStub` is correct here. For v1 rooms, the "stub" doesn't contain the event id, which seems critical here. Also seems like a bug if Synapse leaves out the room IDs but the spec doesn't mention anything like that.